tests/validation/tests/v3_api/test_app.py

```diff
@@ -581,24 +582,29 @@ def create_project_client(request):
         rbac_data["users"][CLUSTER_MEMBER]["user"]
     user_token["user_c1_member"]["token"] = \
         rbac_data["users"][CLUSTER_MEMBER]["token"]
-    client, clusters = get_user_client_and_cluster_app()
+    # create a cluster
+    node_roles = [["controlplane"], ["etcd"],
+                  ["worker"], ["worker"], ["worker"]]
+    cluster_list = []
+    cluster, aws_nodes = create_and_validate_custom_host(node_roles, True)
+    client, cluster_existing = get_user_client_and_cluster()
     admin_client = get_admin_client()
-    cluster1 = clusters[0]
-    cluster2 = clusters[1]
-    assert len(clusters) > 0
+    cluster_list.append(cluster_existing)
+    cluster_list.append(cluster)
+    assert len(cluster_list) > 0
```

Comment:
assert should be for cluster length to be more than 1

Reply:
Done